DigitalLearningSolutions.Data/Services/SectionContentService.cs

```diff
@@ -134,6 +134,7 @@ GROUP BY
                     }
                     else if (section.DiagnosticStatus)
                     {
+                        sectionContent.DiagnosticStatus = section.DiagnosticStatus;
```

Comment:
I don't think this will work. This way the diagnostic status will reflect the diagnstic status of the last tutorial in the section which is not what you want. In addition to fixing this we should include a test which would fail with this implementation (i.e. one where the last diag status is 0 but at least one other tutorial has diag status 1) 🙂

Reply:
That test case does currently exist, I've tested where all statuses are 0 except the second tutorial (out of 4 tutorials). As this is an `else if (section.DiagnosticStatus)` it means that if the diagnostic status is being overwritten, it's only being overwritten with "true"

Reply:
Ah I see! I hadn't considered that the `if` was checking the same variable. That seems fine then 🙂